addons/web/static/src/scss/primary_variables.scss

```diff
@@ -70,6 +70,19 @@ $o-grays: (
     900: $o-gray-900,
 ) !default;
 
+$o-transparent-grays: (
+    'black-15': rgba(black, 0.15),
+    'black-25': rgba(black, 0.25),
+    'black-50': rgba(black, 0.5),
+    'black-75': rgba(black, 0.75),
+    'white-25': rgba(white, 0.25),
+    'white-50': rgba(white, 0.5),
+    'white-75': rgba(white, 0.75),
+    'white-85': rgba(white, 0.85),
+) !default;
```

Comment:
I did not look at the rest of the PR, but first: I doubt that those transparent grays should become a `web` concept, especially not leaving how it worked before somewhere as dead code (I don't see any code being moved/removed in this PR, although those are classes that existed before).